content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -1204,7 +1204,7 @@ Example Usage:
 
 | CEL                                     | Description |
 |-----------------------------------------|-------------|
-| `self.foo == "foo" || (oldSelf.hasValue() && oldSelf.value().foo != "foo")` | Ratcheted rule. Once a value is set to "foo", it must stay foo. But if it existed before the "foo" constraint was introduced, it may use any value |
+| self.foo == "foo" || (oldSelf.hasValue() && oldSelf.value().foo != "foo")` | Ratcheted rule. Once a value is set to "foo", it must stay foo. But if it existed before the "foo" constraint was introduced, it may use any value |
```

Comment:
I don't think this is right. The closing backtick is after `!= "foo")` and now the pipes will be interpreted as table column separators.